internal/server/v2/observation/contained_in.go

```diff
@@ -66,6 +67,43 @@ func trimDirectResp(resp *pbv2.ObservationResponse) *pbv2.ObservationResponse {
 	return result
 }
 
+// For mocking in tests.
+var (
+	getPlacesIn = placein.GetPlacesIn
+	fetchRemote = fetchRemoteWrapper
+)
+
+func fetchRemoteWrapper(
+	metadata *resource.Metadata,
+	httpClient *http.Client,
+	apiPath string,
+	remoteReq *pbv2.NodeRequest,
+) (*pbv2.NodeResponse, error) {
+	remoteResp := &pbv2.NodeResponse{}
+	return remoteResp, util.FetchRemote(metadata, httpClient, apiPath, remoteReq, remoteResp)
```

Comment:
I feel it's better to make this explicit. Will be helpful for debugging if error occurs.

err := util.FetchRemote(metadata, httpClient, apiPath, remoteReq, remoteResp)
if err != nil {
    return nil, err
}
return remoteResp, nil

Reply:
Done.